rclcpp/src/rclcpp/node.cpp

```diff
@@ -126,7 +170,7 @@ Node::Node(
       options.parameter_overrides(),
       options.start_parameter_services(),
       options.start_parameter_event_publisher(),
-      options.parameter_event_qos(),
+      get_parameter_events_qos(*node_base_, options),
```

Comment:
It would be nice, either here or in the documentation of this function being called to indicate what we're doing (getting the QoS overrides from the parameter overrides directly) and why (because we need them before parameters can be bootstrapped).

Reply:
Done in https://github.com/ros2/rclcpp/pull/1532/commits/04ce4210e25566c694f6abbdd68586ed5905b4d2